ansible/requirements.yml

```diff
@@ -100,7 +100,7 @@
   version: 1.1.0
 
 - src: openmicroscopy.postgresql
-  version: 2.0.0
+  version: 3.0.0-m2
```

Comment:
Since this is the first production IDR version with these, are we getting close to dropping `m` then?

Reply:
👍  AFAIK no-ones complained about https://github.com/openmicroscopy/ansible-role-postgresql/tree/3.0.0-m2

Reply:
Now we've deployed non milestone versions, should we move this to `3.0.0` and omero-server to `2.0.1`?

Reply:
Not now, it's already deployed. Next release?

Reply:
Given the effective difference between `3.0.0-m2` and `3.0.1`, agreed I would not block IDR 0.4.4 for this.

Updating our requirements to consume the latest non-milestone role tags before we spin up a new idr-testing server would certainly be valuable.